threatstream.json

```diff
@@ -6224,7 +6224,169 @@
                     "description": "Type of observable to import",
                     "data_type": "string",
                     "order": 1,
-                    "required": true
+                    "required": true,
+                    "value_list": [
```

Comment:
These options consist of itype options and threat_type options. This corresponds to the README:
![image](https://user-images.githubusercontent.com/82124762/144701823-18a55ce9-4741-4258-b974-037129ad4f3d.png)

Reply:
With this, users won't run into typo issues, since the options are taken directly from the dropdown.